lib/queryBuilder/QueryBuilder.js

```diff
@@ -365,12 +367,63 @@ class QueryBuilder extends QueryBuilderBase {
     return this._resultModelClass || this.modelClass();
   }
 
+  isFind() {
+    return (
+      !this._explicitRejectValue &&
```

Comment:
@koskimas why is this check for `!this._explicitRejectValue` here? I've removed it to see if things change, and all tests still pass...

Reply:
I have no idea. Let's remove it if no test fail